src/test/java/teammates/test/util/Url.java

```diff
@@ -0,0 +1,16 @@
+package teammates.test.util;
+
+import teammates.test.driver.TestProperties;
+
+public class Url extends teammates.common.util.Url {
```

Comment:
Can you add header comments to both parent and child classes? Even if we give different names, it may not be clear how they differ.